docs/integrations/webhooks/netlify.md

```diff
@@ -42,10 +148,37 @@ Follow the below steps to configure the Netlify webhook.
 7. Click **Save**.
 8. Verify Netlify events are getting ingested in Sumo Logic by executing the following query on Sumo Logic's Log Search panel.
 ```sql
-`_sourceCategory=webhook/netlify`
+`_sourcecategory=webhook/netlify`
 ```
 
 :::info
 - For detailed information about webhook creation, refer to the [Netlify Documentation](https://docs.netlify.com/site-deploys/notifications/#outgoing-webhooks).
 - For support, [contact Netlify](https://www.netlify.com/support/).
 :::
+
+### Installing the Netlify app
+This section has instructions for installing the Sumo Logic app for Stripe.
+{@import ../../reuse/apps/app-install-v2.md}
```

Comment:
@priyansh-sumo, the build request check is failing because the path to the `app-install-v2.md` file isn't correct. Could you correct it?

Reply:
Added files